quarkus/src/main/resources/postman.collections/JavaOpenCommerce.postman_collection.json

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
This file, IMHO should be in `src/test/resources`. It shouldn't be attached to build jar.